daemon/api.go

```diff
@@ -738,7 +738,7 @@ func snapUpdateMany(inst *snapInstruction, st *state.State) (msg string, updated
 		return "", nil, nil, err
 	}
 
-	updated, tasksets, err = snapstateUpdateMany(st, inst.Snaps, inst.userID)
+	updated, tasksets, err = snapstateUpdateMany(st, inst.Snaps, inst.userID, assertstate.ValidateRefreshes)
```

Comment:
As discussed online, let's go with a global `ValidateRefreshes` function in `snapstate`, which is set by `assertstate` on an `init()` function when it's imported.